facesign-service/server.ts

```diff
@@ -49,8 +49,16 @@ app.post("/session-token", async (req, res) => {
 app.post("/login", async (req, res) => {
   let faceSignUserId: string = crypto.randomUUID();
 
-  const { faceScan, key, userAgent, auditTrailImage, lowQualityAuditTrailImage, sessionId } =
-    req.body;
+  const {
+    faceScan,
+    key,
+    userAgent,
+    auditTrailImage,
+    lowQualityAuditTrailImage,
+    sessionId,
+    groupName = GROUP_NAME,
+    faceVector = true,
```

Comment:
We should control this from inside the service, I'd argue. Mixed feelings, tho, since it'll make an app in an enclave more complicated. However, letting this be caller's choice feels too easy to misuse.